tests/e2e/tests/e2e_happy_path/HappyPath.spec.ts

```diff
@@ -115,22 +115,22 @@ suite('Language server validation', async () => {
     });
 
     test('Error highlighting', async () => {
-        const textForErrorDisplaying: string = '$#%@#';
-
+        await driverHelper.getDriver().sleep(TimeoutConstants.TS_SUGGESTION_TIMEOUT);   // workaround https://github.com/eclipse/che/issues/19004
```

Comment:
I'm just thinking here - Are both of these workarounds needed here?
By both workaround I mean this initial sleep and also the possible refresh of the page on lines 126-130.

I'm not blocking the PR becuase of this of course... just food for thoughts.

Reply:
I had just followed investigation results from here https://github.com/eclipse/che/issues/19004#issuecomment-777808178
`sleep()` has made error highlighting verification quite stable.